scripts/install.sh

```diff
@@ -107,10 +107,9 @@ main() {
   fi
 
   # Enabling statements for ~/.zshrc
-  msg="
-  # Set Spaceship ZSH as a prompt
-  autoload -U promptinit; promptinit
-  prompt spaceship"
+  msg="\n# Set Spaceship ZSH as a prompt"
+  msg+="\nautoload -U promptinit; promptinit"
+  msg+="\nprompt spaceship"
```

Comment:
Hi can you explain why we change these lines ?

Reply:
> I have also removed some blank characters in enabling statements on install.sh script, otherwise, the uninstall script doesn't remove the enabling statements, since it doesn't...

Today the uninstall script doesn't handle white spaces, so I remove them from install.sh. This is the motivation of these changes.